website/pages/en/cookbook/avoid-eth-calls.mdx

```diff
@@ -100,3 +100,17 @@ Note: Declared eth_calls can only be made in subgraphs with specVersion >= 1.2.0
 ## Conclusion
 
 We can significantly improve indexing performance by minimizing or eliminating `eth_calls` in our subgraphs.
+
+## Subgraph Best Practices 1-6
+
+1.  [Improve Query Speed with Subgraph Pruning](https://thegraph.com/docs/en/cookbook/pruning/)
```

Comment:
Yes, because absolute links have the language (English) hardcoded in them, whereas relative ones keep the current language (and the `href` is copied over as-is to translated pages).